web/app/scripts/src/services/fieldsets-parser.es.js

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ * This module parses a form API into an angular-formly array
+ *
+ */
+
+import { angular } from '../libraries';
+
+import FieldsConfigModule from 'fields-config';
+
+export const moduleProperties = {
+    moduleName: 'incuna-surveys.field-parser',
+    componentName: 'FieldsParser'
+};
+
+const module = angular.module(moduleProperties.moduleName, [
+    FieldsConfigModule.moduleName
+]);
+
+module.service(moduleProperties.componentName, [
```

Comment:
If returning an object of methods, use `factory`. If you use `service`, you can assign each method to `this` instead of onto an object: less nesting and no `return`.